bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js

```diff
@@ -0,0 +1,19 @@
+import { check } from 'meteor/check';
+import emitter from '../emitter';
+
+export default function ({ body }, meetingId) {
```

Comment:
How can we make sure it doesn't run in both servers, looks for me the back end and front end will emit it

Reply:
Actually, I think that only back end will emit it, as the `UpdateBreakoutRoomsTimeEvtMsg` message comes through the `from-akka-apps-redis-channel` channel from akka. It is still a problem, of course, as we need the front end to emit it. I'm working on a temporary solution as a fancier solution is being discussed (https://github.com/bigbluebutton/bigbluebutton/issues/14493).